src/tests/performance/ping_pong.rs

```diff
@@ -110,10 +107,12 @@ async fn p001_t1_PING_PONG_throughput() {
 
         let mut synth_handles = Vec::with_capacity(synth_count);
         let test_start = tokio::time::Instant::now();
-        for _ in 0..synth_count {
-            synth_handles.push(tokio::spawn(simulate_peer(node_addr)));
+        for i in 0..synth_count {
+            synth_handles.push(tokio::spawn(simulate_peer(node_addr, i + used)));
         }
 
+        used += synth_count;
```

Comment:
would this be more efficient:
```suggestion
        for i in 0..synth_count {
            synth_handles.push(tokio::spawn(simulate_peer(node_addr, used)));
            used += 1;
        }
```

Reply:
Done.